automated/linux/imagenet-resnet50/imagenet-resnet50.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+
+usage() {
+    echo "Usage: $0 [-a <home-directory>]
+                    [-m <mount-directory>]
```

Comment:
[-t <true|false>] 
Do you need this in usage() ?

Reply:
Thanks for catching this, resolved.